packages/imperative/src/profiles/src/utils/ProfileIO.ts

```diff
@@ -312,7 +312,7 @@ export class ProfileIO {
     private static crashInTeamConfigMode() {
         if (ImperativeConfig.instance.config?.exists) {
             try {
-                throw new Error("A Zowe V1 profile operation was attempted with a Zowe V2 configuration in use.");
+                throw new Error("A Zowe V1 profile operation was attempted with a Zowe team configuration in use.");
```

Comment:
There is no official policy from Rose, Adam, and Ana on the terminology that we should use for our configuration. 
In V2, we needed to differentiate between our old and new configuration, the terms "V1 profiles" and "Team Configuration" made sense. Since we should be removing our references to V1 profiles, I decided to use our command output (whose text is not the primary focus of policy makers) to refer to our one and only available configuration as the "Zowe client configuration". I made an exception in the convert-profiles command to differentiate that we are converting V1 profiles to something distinctively different. Thus I kept the term "Team Configuration" in some output of that particular command. If the majority of people feel that my choice of terminology is not desired, we could revise the command output across the board in our next story.

The particular case that you sited was in ProfileIO.ts. That file was not overhauled in this PR. It will be the primary focus of the next story on V1 profiles. It is possible that we may remove that exception, maybe the entire function (I do not know for sure yet). However we resolve it, this specific message should be addressed during the next V1 story.